csharp-linq/LINQInnerJoin/LinqInnerJoin/EnrolmentHelper.cs

```diff
@@ -0,0 +1,169 @@
+﻿namespace LinqInnerJoin
+{
+    public static class EnrolmentHelper
+    {
+        public static IEnumerable<dynamic> GetEnrollments(IEnumerable<Enrolment> enrollments)
+        {
+            var students = Student.GetDummyStudents();
+            var courses = Course.GetDummyCourses();
+
+            var enrollements = enrollments  //Enrolment data source                            
+                               .Join(   // First Join - Joining Student with Enrolment
+                                   students,
+                                   enrolment_lv1 => enrolment_lv1.StudentId,
+                                   student => student.Id,
+                                   (enrolment_lv1, student) => new { enrolment_lv1, student }   // result from first join
+                                )
+                               .Join(   // Second Join - Joining Course with the result of first join 
+                                   courses,
+                                   enrolment_lv2 => enrolment_lv2.enrolment_lv1.CourseId,
+                                   course => course.Id,
+                                   (enrolment_lv2, course) => new { enrolment_lv2, course }  // result fromn second join
+                                )
+                               .Select(e => new      // Selecting result from the selection of two joins
+                               {
+                                   e.enrolment_lv2.enrolment_lv1.Id,
+                                   StudentName = e.enrolment_lv2.student.Name,
+                                   CourseName = e.course.Name,
+                               });
```

Comment:
Use lowerCamelCase to name local variables instead of snake casing : 
enrolmentLv1 or enrolmentLevel1 instead of enrolment_lv1

Comments are not needed since you explained this section pretty well in the article (they just clutter the code) 
Or you can use a intermediate variable to hold join1 and join2 results and name them respectively for clarity.

Notice that enrolment and Enrollment are both valid words but we need to choose one for consistency across the code and the article (british vs american) :)

Reply:
did the fix, updated the variable names and used enrolment now everywhere